### PR TITLE
Add key shortcuts to search result frame and main table

### PR DESCRIPTION
**Pull request by bartsch-dev**
- <kbd>Pos1</kbd> / <kbd>HOME</kbd> now select the first/last entry in the main table and the search result frame.
- <kbd>UP</kbd> / <kbd>Down</kbd> / <kbd>Tab</kbd> / <kbd>shift+Tab</kbd> in the search result frame have now the same functionality as in the main  table.

### CODE REVIEW

File: src/main/java/net/sf/jabref/gui/search/SearchResultFrame.java [hunk 1]
```diff
@@ -172,6 +172,54 @@ public void actionPerformed(ActionEvent e) {
...
+        String selectLast = "selectLast";
+        im.put(Globals.getKeyPrefs().getKey(KeyBinding.SELECT_LAST), selectLast);
+        am.put(selectLast, new AbstractAction() {
+            @Override
+            public void actionPerformed(ActionEvent event) {
+                selectLastEntry();
+            }
+        });
+
+
```
Comment (boceckts) [inline src/main/java/net/sf/jabref/gui/search/SearchResultFrame.java]: Remove empty line.

File: src/main/java/net/sf/jabref/gui/search/SearchResultFrame.java [hunk 2]
```diff
@@ -172,6 +172,54 @@ public void actionPerformed(ActionEvent e) {
         im.put(Globals.getKeyPrefs().getKey(KeyBinding.CLOSE_DATABASE), "close");
         am.put("close", closeAction);
 
+
```
Comment (boceckts) [inline src/main/java/net/sf/jabref/gui/search/SearchResultFrame.java]: Remove empty line.

**Comment by boceckts** [comment]
Add the added localization keys to the localization files.

**Comment by boceckts** [comment]
Looks good to me, only localization is missing. 👍

File: src/main/java/net/sf/jabref/gui/search/SearchResultFrame.java (hunk 1 above)
Comment (bartsch-dev) [inline src/main/java/net/sf/jabref/gui/search/SearchResultFrame.java]: fixed

File: src/main/java/net/sf/jabref/gui/search/SearchResultFrame.java (hunk 2 above)
Comment (bartsch-dev) [inline src/main/java/net/sf/jabref/gui/search/SearchResultFrame.java]: fixed

**Comment by bartsch-dev** [comment]
Fixed localization

File: CHANGELOG.md [hunk 3]
```diff
@@ -30,6 +30,8 @@ We refer to [GitHub issues](https://github.com/JabRef/jabref/issues) by using `#
 - `note` field is now exported as `comments` field in MS-Office 2007 xml format
 - `comments` field in MS-Office 2007 xml format is now imported as `note` field
 - [#463](https://github.com/JabRef/jabref/issues/463): Disable menu-item and toolbar-buttons while no database is open
+- `Pos1`/`HOME` now select the first/last entry in the main table and the search result frame.
```
Comment (koppor) [inline CHANGELOG.md]: Could you render them using `<kbd>key</kbd>` please?

File: src/main/java/net/sf/jabref/gui/maintable/MainTable.java [hunk 4]
```diff
@@ -170,6 +172,7 @@ public MainTable(MainTableFormat tableFormat, MainTableDataModel model, JabRefFr
 
         //Override 'selectNextColumnCell' and 'selectPreviousColumnCell' to move rows instead of cells on TAB
         ActionMap am = getActionMap();
```
Comment (koppor) [inline src/main/java/net/sf/jabref/gui/maintable/MainTable.java]: could you rename that to `actionMap` and `im` to `inputMap`?

File: src/main/java/net/sf/jabref/gui/maintable/MainTable.java [hunk 5]
```diff
@@ -194,6 +197,24 @@ public void actionPerformed(ActionEvent e) {
                 panel.selectPreviousEntry();
             }
         });
+
+        String selectFirst = "selectFirst";
+        im.put(Globals.getKeyPrefs().getKey(KeyBinding.SELECT_FIRST_ENTRY), selectFirst);
+        am.put(selectFirst, new AbstractAction() {
```
Comment (koppor) [inline src/main/java/net/sf/jabref/gui/maintable/MainTable.java]: Can this be written as lambda?

File: src/main/java/net/sf/jabref/gui/maintable/MainTable.java [hunk 6]
```diff
@@ -194,6 +197,24 @@ public void actionPerformed(ActionEvent e) {
...
+            @Override
+            public void actionPerformed(ActionEvent event) {
+                panel.selectFirstEntry();
+            }
+        });
+
+        String selectLast = "selectLast";
+        im.put(Globals.getKeyPrefs().getKey(KeyBinding.SELECT_LAST_ENTRY), selectLast);
+        am.put(selectLast, new AbstractAction() {
```
Comment (koppor) [inline src/main/java/net/sf/jabref/gui/maintable/MainTable.java]: Can this be written as lambda?

**Comment by koppor** [comment]
Functionality works as expected. Some minor code comments remain.

File: src/main/java/net/sf/jabref/gui/maintable/MainTable.java (hunk 5 above)
Comment (Siedlerchr) [inline src/main/java/net/sf/jabref/gui/maintable/MainTable.java]: I know that this screams lambda, but AbstracAction unfortunately does not have an functional Interface... :disappointed: 
Got fooled by it already ;)
http://blog.codefx.org/techniques/getting-rid-of-anonymous-classes/

File: src/main/java/net/sf/jabref/gui/maintable/MainTable.java (hunk 4 above)
Comment (bartsch-dev) [inline src/main/java/net/sf/jabref/gui/maintable/MainTable.java]: fixed

File: CHANGELOG.md (hunk 3 above)
Comment (bartsch-dev) [inline CHANGELOG.md]: fixed

**Comment by bartsch-dev** [comment]
Renamed variables (as already said the actions cannot be shortened with lambdas).

**Pull request merged**